client.js

```diff
@@ -45,101 +41,335 @@ bot.on('message', function (message) {
+                            const diffDuration = moment.duration(diff)
+                            const snoozeExpiration = moment(watch.expiration).add(0, 'seconds');
+                            const snoozeDiff = snoozeExpiration.diff(now)
+                            const snoozeDuration = moment.duration(snoozeDiff)
+                            const price = watch.price == -1 ? 'No Price Criteria' : watch.price.toString().concat('pp')
+                            const item = formatCapitalCase(watch.name)
+                            const server = `${formatCapitalCase(watch.server)} Server`
+                            watches.push({
+                                name: `${formatCapitalCase(watch.name)} | ${price} | ${formatCapitalCase(watch.server)}`,
+                                value: `Expires in ${diffDuration.days()} days ${diffDuration.hours()} hours  and ${diffDuration.minutes()} minutes`,
+                                inline: false
+                            })
+
+                            if (watch.snoozed) {
+                                watches.push({
+                                    name: `💤 💤 💤 💤  💤  💤 💤 💤 💤 💤  💤`,
+                                    value: `Snoozed for another ${snoozeDuration.hours()} hours and ${snoozeDuration.minutes()} minutes`,
+                                    inline: false
+                                })
+                            }
+                            message.author.send(
+                                new Discord.MessageEmbed()
+                                .setColor(SERVER_COLOR[watch.server])
+                                .setAuthor(`${formatCapitalCase(watch.name)}`, url, `https://wiki.project1999.com/${formatItemNameForWiki(watch.name)}`)
+                                .addFields(watches)
+                                .setFooter(`To snooze this watch for 6 hours, click 💤\nTo end this watch, click ❌\nTo extend this watch, click ♻`)
+                            )
+                            .then((message) => {
+                                message
+                                .react('💤')                     // for "snooze watch"
+                                .then(() => message.react('❌')) // for "delete watch"
+                                .then(() => message.react('♻'))  // for "extend watch"
+                                .then(() => {
+                                    const react_filter = (reaction, user) => {
+                                        if (user.bot) {
+                                            return;
+                                        }
+                                        return reaction.emoji.name === '💤' || reaction.emoji.name === '❌' || reaction.emoji.name === '♻';
+                                    }
+                                    const collector = message.createReactionCollector(react_filter, { time: 1000 * 60 * 60 * 24 , dispose: true});
```

Comment:
Using emojis in other commands too now, it's a much nice experience.  I'm wondering if the 24 hour period on these event listeners will be expensive cpu-wise...